physics/equipotential_test.cpp

```diff
@@ -307,18 +310,83 @@ TEST_F(EquipotentialTest, BodyCentredBodyDirection_EquidistantEnergies) {
 TEST_F(EquipotentialTest, BodyCentredBodyDirection_GlobalOptimization) {
   mathematica::Logger logger(TEMP_DIR / "equipotential_bcbd_global.wl",
                              /*make_unique=*/false);
-  std::int64_t const number_of_days = 30;
+  std::int64_t const number_of_days = 100;
+  auto const earth = solar_system_->massive_body(
+      *ephemeris_, SolarSystemFactory::name(SolarSystemFactory::Earth));
+  auto const moon = solar_system_->massive_body(
+      *ephemeris_, SolarSystemFactory::name(SolarSystemFactory::Moon));
   auto const dynamic_frame(
       BodyCentredBodyDirectionDynamicFrame<Barycentric, World>(
-          ephemeris_.get(),
-          solar_system_->massive_body(
-              *ephemeris_,
-              SolarSystemFactory::name(SolarSystemFactory::Earth)),
-          solar_system_->massive_body(
-              *ephemeris_,
-              SolarSystemFactory::name(SolarSystemFactory::Moon))));
+          ephemeris_.get(), moon, earth));
   CHECK_OK(ephemeris_->Prolong(t0_ + number_of_days * Day));
 
+  DegreesOfFreedom<Barycentric> const earth_dof =
+      ephemeris_->trajectory(earth)->EvaluateDegreesOfFreedom(t0_);
+  DegreesOfFreedom<Barycentric> const moon_dof =
+      ephemeris_->trajectory(moon)->EvaluateDegreesOfFreedom(t0_);
+  KeplerOrbit<Barycentric> const moon_orbit(
+      *earth,
+      *moon,
+      moon_dof - earth_dof,
+      t0_);
+  KeplerianElements<Barycentric> const moon_elements =
+      moon_orbit.elements_at_epoch();
+
+  KeplerianElements<Barycentric> const elements{
+      .periapsis_distance = 7000 * Kilo(Metre),
+      .apoapsis_distance = 0.8 * *moon_elements.periapsis_distance,
+      .inclination = moon_elements.inclination,
+      .longitude_of_ascending_node = moon_elements.longitude_of_ascending_node,
+      .argument_of_periapsis = *moon_elements.argument_of_periapsis + Degree,
+      .mean_anomaly = 0 * Degree};
+  auto const earth_world_dof = dynamic_frame.ToThisFrameAtTime(t0_)(earth_dof);
+  auto const moon_world_dof = dynamic_frame.ToThisFrameAtTime(t0_)(moon_dof);
+  Position<World> const qearth = earth_world_dof.position();
+  Position<World> const qmoon = moon_world_dof.position();
+  Velocity<World> const vearth = earth_world_dof.velocity();
+  Velocity<World> const vmoon = moon_world_dof.velocity();
+  std::vector<DegreesOfFreedom<Barycentric>> const initial_states{
```

Comment:
Please comment what's going on in this test, in particular the test trajectories.